### ISSUE TITLE
Dark mode (or "night mode")

### ISSUE THREAD

**Issue by LoadingBG** [exercism/exercism#5436]
As every programmer has used night mode for IDEs and apps, I think night mode would be a great feature for Exercism. Also it would look really nice with the blue-ish theme you chose.

**Comment by iHiD** [comment]
This is on the roadmap.

**Comment by ErikSchierboom** [comment]
Hello 👋

With the launch of Exercism v3, we are **closing all issues in this repository** to help give us a clean slate to detect new problems. If this issue is still relevant to Exercism v3 (e.g. it's a feature that we haven't implemented in v3, or a bug that still exists), please reopen it and we will review it and post an update on it as soon as we get chance.

Thanks for helping make Exercism better, and we hope you enjoy v3 🙂

**Comment by armahillo** [comment]
![Screenshot 2024-06-13 at 1 34 38 PM](https://github.com/exercism/exercism/assets/502363/0622884e-3bb7-4c8c-acd0-3a2070bc0a1d)

This isn't okay. Dark mode is an a11y issue, not just an aesthetic choice.

Websites with mostly white backgrounds are actually harder for me to read, especially at length. At night, particularly, it is physically painful for me to look at sites that have bright backgrounds.

I don't care if you want to allow different color schemes and put that under "Insiders" but a trivial dark mode shouldn't be paywalled.

**Comment by kotp** [comment]
> a trivial dark mode shouldn't be paywalled.

We agree, and the trivial dark mode is not paywalled.  I just created an account to test this as well, and I am not getting the same results as you, perhaps it is a browser setting or perhaps a browser that does not honor the "system settings" for dark mode.

Can you describe your setup?

**Comment by armahillo** [comment]
@kotp OK I was able to find it by going to Settings -> Preferences -> Theme, thanks! This is much easier on my eyes (Reminds me of old EGA games :joy:). I was confused because the toggle for "dark mode" on the home page (in my screenshot) is where I would have expected to see this. 

Perhaps since I don't _have_ the fancy darkmode available, this switch could link directly to the preferences theme settings where the different options are available?  Since I found "switch for enabling darkmode" on the homepage, I had no reason to think there might be a different dark mode available buried in the settings page.

**Comment by kotp** [comment]
> Perhaps since I don't _have_ the fancy darkmode available, this switch could link directly to the preferences theme settings where the different options are available? Since I found "switch for enabling darkmode" on the homepage, I had no reason to think there might be a different dark mode available buried in the settings page

Because of the initial (understandable violent) reaction, perhaps we need to address this.

I would suggest an "accessibility" icon with alt text, that indicates on the face of the website so that this is not hard to find, and easily togglable?  @iHiD @ErikSchierboom ?

![darkmode](https://github.com/exercism/exercism/assets/16747/167dd5dc-1afb-48bf-b7dd-adbbf2417dd9)

This is where I would expect to "see" it.

From descriptions, I think "yellowish ball/toggle".

**Comment by gumbarros** [comment]
> Websites with mostly white backgrounds are actually harder for me to read, especially at length. At night, particularly, it is physically painful for me to look at sites that have bright backgrounds.

Yes, it's unfair with people from countries that can't afford 10 dollars per month.
